src/Service/CheckRecaptchaV2.php

```diff
@@ -34,6 +34,7 @@ public function check($challenge, $response)
             curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
             curl_setopt($curl, CURLOPT_TIMEOUT, app('config')->get('recaptcha.options.curl_timeout', 1));
             curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, false);
+            curl_setopt($curl, CURLOPT_PROXY, app('config')->get('recaptcha.options.curl_proxy', NULL));
```

Comment:
Thank you for pointing that out. I had no idea.

Edited the code, but it is currently untested. I will test shortly and make a note here after confirming that it works as expected.